recipe/meta.yaml

```diff
@@ -22,11 +24,9 @@ requirements:
     - toolchain
     - cython
     - numpy x.x
-    - numpy >=1.6
   run:
     - python
     - numpy x.x
-    - numpy >=1.6
```

Comment:
That is fine to keep it. But in my mind numpy 1.6 is so old that I don't even consider it as a possibility. Do you want me to restore it?